packages/autocomplete-plugin-recent-searches/src/index.ts

```diff
@@ -0,0 +1 @@
+export { createPlugin as default } from './createPlugin';
```

Comment:
Unsure about exporting the plugin as `default` — I think we should go for named exports.

Reply:
https://github.com/algolia/autocomplete.js/pull/316/commits/17809ca4d7b7632c6a872298771cd2c90b42065d